### PR TITLE
Fix issue #530: Explaining `fetch` command via the word "pull" may confuses the newbie

### PR DESCRIPTION
**Pull request by YueLinHo**

### CODE REVIEW

File: book/02-git-basics/sections/remotes.asc [hunk 1]
```diff
@@ -111,7 +111,7 @@ After you do this, you should have references to all the branches from that remo
 
 If you clone a repository, the command automatically adds that remote repository under the name ``origin''.
 So, `git fetch origin` fetches any new work that has been pushed to that server since you cloned (or last fetched from) it.
-It's important to note that the `git fetch` command pulls the data to your local repository – it doesn't automatically merge it with any of your work or modify what you're currently working on.
+It's important to note that the `git fetch` command just fetches the data to your local repository – it doesn't automatically merge it with any of your work or modify what you're currently working on.
```
Comment (ben) [inline book/02-git-basics/sections/remotes.asc]: Instead of "just fetches", let's say "only downloads".

Comment (YueLinHo) [inline book/02-git-basics/sections/remotes.asc]: Thanks. ^________^

**Comment by YueLinHo** [comment]
Updated with amend.

**Comment by ben** [comment]
Lovely. Thanks!

**Comment by YueLinHo** [comment]
Thanks for merging. ^__^

**Pull request merged**